### PR TITLE
Quipu staking

### PR DESCRIPTION
**Pull request by letier3110**
https://www.notion.so/madfissolutions/Display-Quipu-APY-2a65d5d961b445328c99dfcfd8770fe9

### CODE REVIEW

File: src/screens/wallet/token-list/token-list-item/use-quipu-apy.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        }
+        quipuApyMem.value = apy;
+        quipuApyMem.timestamp = Date.now().toString();
+        setQuipuApy(apy);
+      } catch {}
+    })();
+  }, []);
+
+  return { apy: quipuApy, formattedApy: quipuApy.toFixed(2).replace(/[.,]00$/, '') };
+};
```
Comment (unixvb) [inline src/screens/wallet/token-list/token-list-item/use-quipu-apy.ts:96]: move this api call logic into state & epic

File: src/utils/quipu-apy.util.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    level: number;
+    timestamp: string;
+  };
+}
+
+export const getQuipuStakingInfo = async (): Promise<QuipuswapStakingInfoResponse> => {
+  return quipuStakingApi.get('/list/3').then(r => r.data);
+};
+
+export const loadQuipuApy$ = () =>
```
Comment (LendiHop) [inline src/utils/quipu-apy.util.ts:62]: maybe just "export const loadQuipuApy$ = from(getQuipuStakingInfo()).pipe(...." ?

File: src/store/wallet/wallet-epics.ts [hunk 3]
```diff
@@ -45,6 +47,17 @@ const updateDataActions = () => [
   loadSelectedBakerActions.submit()
 ];
 
+const loadQuipuApyEpic = (action$: Observable<Action>) =>
+  action$.pipe(
+    ofType(loadQuipuApyActions.submit),
+    switchMap(() =>
+      loadQuipuApy$().pipe(
```
Comment (LendiHop) [inline src/store/wallet/wallet-epics.ts:54]: and here just "loadQuipuApy$.pipe(..."

**Review by LendiHop — APPROVED** [review]

File: src/navigator/main-stack.tsx [hunk 4]
```diff
@@ -74,6 +75,7 @@ export const MainStackScreen = () => {
     dispatch(loadTokenBalancesActions.submit());
     dispatch(loadActivityGroupsActions.submit());
     dispatch(loadSelectedBakerActions.submit());
+    dispatch(loadQuipuApyActions.submit());
```
Comment (unixvb) [inline src/navigator/main-stack.tsx:78]: should this be refreshed every 60 secs?
 lets do this only on app init

**Review by unixvb — APPROVED** [review]

**Pull request merged**